.github/scripts/generate-release-body.ts

```diff
@@ -209,13 +212,16 @@ async function main() {
 
   const clientPRs = prByLabels[CLIENT_CHANGES_LABEL] || [];
   const runtimePRs = prByLabels[RUNTIME_CHANGES_LABEL] || [];
+  const testsPRs = prByLabels[TESTS_CHANGES_LABEL] || [];
+  const ciPRs = prByLabels[CI_CHANGES_LABEL] || [];
+  const otherPRs = prByLabels[OTHER_CHANGES_LABEL] || [];
```

Comment:
`other = everything - runtime - client` won't work because this will cause some of the pr which have more than 1 label to be included multiple times.
(example: let's suppose a Pr (123) has 3 tags  `shiden`, `runtime` and `shibuya`) 
This pr will be included in runtime and 2 times in other.
Of course this can be managed but that would not be a simpler solution (which is our objective).
Current solution is good enough for now imo.

Reply:
Yes, it's possible for a single PR to have multiple labels and that is fine.
It was also possible before.

The example I gave will work, not sure how you see it not working?
Iterate over all PRs and make sure that it's not included in neither of the `client` or `runtime` lists.

And I'm sorry but I won't approve this - the code highlighted in this comment is a clear anti-pattern.
It has unnecessary complexity, and is not resistant to addition of new labels (which isn't important for this example but it is in general).

Reply:
> It was also possible before.

Earlier only the PRs that  had `runtime`, `client`, `breaksapi` or empty label were included (not all), so if I had put a `test` label, it wouldn't have got included. 
> Iterate over all PRs and make sure that it's not included in neither of the client or runtime lists.

We also have to make sure that a PR doesn't get include twice which has different labels, For example: If a PR has `test` and `ci` label, then we have to make sure it only gets included once.

Now it has all the functionality that you wanted.